### ISSUE TITLE
Backbone

### ISSUE THREAD

**Issue by guybedford** [jspm/registry#190]
I just tried Backbone and it seemed to be loading fine for me in the browser.

@jason can you explain the code you were using that wasn't working. It would be great to look into it.

**Comment by guybedford** [comment]
Tried:

```
jspm install backbone -o "{ format: 'global' }"
```

Then that doesn't work because "_" isn't defined
So we need to ensure underscore is acting like a global
which means

```
jspm install underscore -o "{ format: 'global' }"
```

etc?

**Comment by speigg** [comment]
I'm running into this problem with trying to make an override for another library. I believe the library that is using `_` should have a shim option that lets jspm map '_' to the appropriate dependency. So... something like:

**Comment by speigg** [comment]
Another option is to override the `deps` syntax:

// shim something:
{
    "deps": [
       "three",
       {"_": "underscore"},
       {"$": "jquery"}
    ]
}

**Comment by guybedford** [comment]
@speigg yes this feature is planned at https://github.com/systemjs/systemjs/issues/148.

**Comment by speigg** [comment]
Great!
